tests/unit/_internal/models/test_model.py

```diff
@@ -38,6 +38,14 @@
 """
 
 
+class AdditionClass:
+    def __init__(self, x):
+        self.x = x
+
+    def __call__(self, y):
+        return self.x + y
```

Comment:
```suggestion
class AdditionClass:
    def __init__(self, x: int):
        self.x = x

    def __call__(self, y: int) -> int:
        return self.x + y
```

Should make the type ignore below unnecessary.